### ISSUE TITLE
Staging Review finding: More accessible masking of email addresses

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98442]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Identity
**Product:** sign-in transition
**Feature:** interstitial

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that does not meet best practices.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-meets-best-practices)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When a Login.gov or ID.me account matching the user's email address is found, the email address is displayed on screen with a portion of the address masked using asterisks. Screen reader behavior varies, but as currently coded many will announce that text as something like "vet star star star star star star star star star star star star at gmail dot com" or "vet twelve stars at gmail dot com" or similar.

That's not really ideal, and can be hard for a screen reader user to interpret and parse.

<details><summary>Link, screenshot or steps to recreate</summary>

!["Start using your Login.gov or ID.me account now" section with email addresses redacted with stars.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/91699-Brian-1.png?raw=true)

</details>

## Recommended action

I'd recommend something like this:

```
<p>We found an existing Login.gov account for your email address: <span aria-hidden-"true">vet************@gmail.com</span> <span class="sr-only">starting with vet and ending with @gmail.com</span>.</p>
```

That will be announced as "We found an existing Login.gov account for your email address starting with vet and ending with @gmail.com." That's a closer match to how a human would share this information with another human, and is much easier to make sense of (and I think should be programmatically possible).

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
